src/components/Toolbar.tsx

```diff
@@ -1,37 +1,48 @@
 import DisplacementModal from '@/components/DisplacementModal'
 import FileUploadButton from '@/components/FileUploadButton.tsx'
 import SwitchWithTitle from '@/components/SwitchWithTitle'
+import { useAppDispatch, useAppSelector } from '@/hooks/use-redux.ts'
+import { setDisplayCoordinateAxes, setDisplayNodeIndices } from '@/redux/slices/modelViewSettingSlice.ts'
```

Comment:
the `.ts` file extension is unnecessary

```suggestion
import { useAppDispatch, useAppSelector } from '@/hooks/use-redux'
import { setDisplayCoordinateAxes, setDisplayNodeIndices } from '@/redux/slices/modelViewSettingSlice'
```